src/modules/0.0.18/aws_ecs_simplified/entity/ecs_simplified.ts

```diff
@@ -47,4 +47,15 @@ export class EcsSimplified {
 
   @Column({ nullable: true, })
   loadBalancerDns?: string;
+
+  @Column({
+    default: false,
+  })
+  forceNewDeployment: boolean;
```

Comment:
I am actually thinking to keep things simpler we shouldn't expose this, but set it to `true` in `service` always.